astroquery/hitran/reader.py

```diff
@@ -123,6 +261,43 @@ def quanta_formatter(group_global='class1', group_local='group1'):
 
     return loc,glob
 
+def download_hitran(m, i, numin, numax):
+    """
+    Download HITRAN data for a particular molecule. Based on fetch function from
+    hapi.py.
+
+    Parameters
+    ----------
+    m : int
+        HITRAN molecule number
+    i : int
+        HITRAN isotopologue number
+    numin : real
+        lower wavenumber bound
+    numax : real
+        upper wavenumber bound
+    """
+    iso_id = str(ISO[(m,i)][ISO_INDEX['id']])
+    mol_name = ISO[(m,i)][ISO_INDEX['mol_name']]
+    url = HITRAN_URL + '/lbl/api?' + \
+        'iso_ids_list=' + iso_id + '&' + \
+        'numin=' + str(numin) + '&' + \
+        'numax=' + str(numax)
+    try:
+        req = urllib2.urlopen(url)
+    except urllib2.HTTPError:
+        raise Exception('Failed to retrieve data for given parameters.')
+    except urllib2.URLError:
+        raise Exception('Cannot connect to {0}.'.format(HITRAN_URL))
+    CHUNK = 64 * 1024
+    filename = os.path.join(cache_location, '{0}.data'.format(mol_name))
+    with open(filename, 'w') as fp:
+       while True:
+          chunk = req.read(CHUNK)
+          if not chunk: break
+          fp.write(chunk.decode('utf-8'))
+          log.info('{0} bytes written to {1}'.format(CHUNK, filename))
```

Comment:
Probably better to use a progressbar rather than spit out this verbose information so often